temporalcli/commandsmd/commands.md

```diff
@@ -624,17 +639,25 @@ request.
 
 Information about the Task Queue can be returned to troubleshoot server issues.
 
-`temporal task-queue describe --task-queue=MyTaskQueue --task-queue-type="activity"`
-
 Use the options listed below to modify what this command returns.
 
+Note that without a `--select-*` option the result for the default Build ID will be returned.
+The default Build ID is the one mentioned in the first unconditional Assignment Rule.
+If there is no default Build ID, the result for the unversioned queue will be returned.
+
 #### Options
 
 * `--task-queue`, `-t` (string) - Task queue name. Required.
-* `--task-queue-type` (string-enum) - Task Queue type. Options: workflow, activity. Default: workflow.
-* `--partitions` (int) - Query for all partitions up to this number (experimental+temporary feature). Default: 1.
+* `--task-queue-type` (string[]) - Task queue types considered. If not specified, all types are reported. The current valid queue types are workflow, activity, or nexus.
+* `--select-build-id` (string[]) - Task queue filter based on Build ID.
+* `--select-unversioned` (bool) - Include the unversioned queue.
+* `--select-all-active` (bool) - Include all active versions. A version is active if it had new tasks or polls recently.
+* `--report-reachability` (bool) - Display task reachability information.
+* `--legacy-mode` (bool) - Enable a legacy mode for servers that do not support rules-based worker versioning. This mode only provides pollers info.
```

Comment:
Now I am wondering if we should just move the whole `task-queue describe` to `task-queue describe-legacy` or something

Reply:
I don't want to emphasize too much the legacy mode, the new mode is a superset, and the old one should go away... Also, in the usage stats there are very few current users of `task describe`,  so adding a top level command feels like an overkill... But yes, I agree with you that we should make it very clear the breaking changes in the release notes...

Reply:
:+1: Agreed on overkill, no prob